recipes/common/qtile-modules.yml

```diff
@@ -0,0 +1,16 @@
+modules:
+    - type: rpm-ostree
+      install:
+        - qtile
+        - xdg-desktop-portal-wlr
+        - xdg-desktop-portal-gtk
+        - foot
+        - swaylock
```

Comment:
At this time qtile does not support swaylock, but it seems like there is interest in adding support: https://github.com/qtile/qtile/issues/4976

Reply:
@sgallag-insta interesting, how do we lock the screen then? :smile:

Reply:
Open to suggestions here, but as far as I've been able to tell so far, we don't :) 

Apparently older versions of swaylock used to work, until swaylock deprecated an older locking protocol. Anything that requires `ext-session-lock-v1` is no bueno

Reply:
yeesh. im assuming installing an old swaylock version isn't gonna work because of dep conflicts?

Reply:
Might be worth a shot, I'll see what's involved on that front

Reply:
Downgrade seems to be working well enough